cardano-testnet/src/Testnet/SubmitApi.hs

```diff
@@ -0,0 +1,90 @@
+{-# LANGUAGE DisambiguateRecordFields #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
+{- HLINT ignore "Redundant id" -}
+{- HLINT ignore "Redundant return" -}
+{- HLINT ignore "Use head" -}
+{- HLINT ignore "Use let" -}
+{- HLINT ignore "Redundant <&>" -}
+
+module Testnet.SubmitApi
+  ( SubmitApiConf (..)
+  , withSubmitApi
+  ) where
+
+import           Cardano.Testnet
+
+import           Prelude
+
+import qualified Hedgehog as H
+import qualified Hedgehog.Extras.Test.Base as H
+
+import qualified Testnet.Process.Run as H
+
+import qualified Cardano.Testnet as H
+import           Data.Functor ((<&>))
+import           Hedgehog.Extras (Integration)
+import           Hedgehog.Extras.Stock (Sprocket (..))
+import qualified Hedgehog.Extras.Stock.IO.Network.Sprocket as IO
+import qualified Hedgehog.Extras.Test.Process as H
+import qualified System.IO as IO
+import qualified System.Process as IO
+
+data SubmitApiConf = SubmitApiConf
+  { tempAbsPath :: FilePath
+  , configPath :: FilePath
+  , epochSlots :: Int
```

Comment:
Why isn't the epoch slots being read from the config file?

Reply:
Not sure.  The `cardano-submit-api` CLI requires it.